Dockerfile

```diff
@@ -1,12 +1,26 @@
-FROM osgeo/gdal:ubuntu-full-3.6.3
+FROM osgeo/gdal:ubuntu-full-3.6.3 AS base
+ARG USERNAME
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+
+# Run only if username is set
```

Comment:
Why should we check this?

Reply:
because the vscode devcontainer can set it

Reply:
Is there a problem if vscode set them? (I really don't know, just asking for knowledge)

Reply:
It makes sure that the local files aren't chowned by the dev container. That was an issue.

Reply:
Thank you for the explanation